packages/core/src/btc/transfers.ts

```diff
@@ -24,21 +30,26 @@ export interface TransferPaginationOptions extends PaginationOptions {
   updatedAtLt?: Timestamp;
 }
 
-export interface Transfer {
-  id: string;
-  walletId: string;
-  outputIndex: number;
-  transaction: BtcTransaction | null;
+export interface Transfer
+  extends Omit<
+    TransferDTO,
+    "amount" | "feeAmount" | "confirmation" | "transaction"
+  > {
+  amount: BN;
   feeAmount: BN | null;
+  confirmation: BN;
+  transaction: BtcTransaction | null;
```

Comment:
How about this?
```suggestion
  transaction?: BtcTransaction;
```

Reply:
Bob and Jun made a convention that explicit type forms are better than `?`.